docs/index.asciidoc

```diff
@@ -149,6 +162,7 @@ API key API].
 
 [id="plugins-{type}s-{plugin}-ca_file"]
 ===== `ca_file` 
+deprecated[8.8.0, Replaced by <<plugins-{type}s-{plugin}-ssl_certificate_authorities>>]
```

Comment:
This deprecation version should related the plugin's version and not the Logstash version.
Same for following changes about the `deprecated`